### PR TITLE
unify text style in GUI

### PR DESCRIPTION
**Pull request by kswang1029**
**Description**
This is an attempt of unifying text style elsewhere in the GUI for issue #1931.

Basically we keep the text style in the root menu (eg File), tab title, widget/dialog title as "title-case". Others such as button name, tooltip, placeholder, and switch name are "sentence-case". Hopefully the coverage is complete but if you spot missing parts, please let me know.  

**Checklist**

For linked issues (if there are):
- [x] assignee and label added
- [x] ZenHub issue connection, board status, and estimate updated

For the pull request:
- [x] reviewers and assignee added
- [x] ZenHub estimate, milestone, and release (if needed) added
- [x] ~e2e test passing~ / corresponding fix added
- [x] ~changelog updated~ / no changelog update needed
- [x] ~protobuf updated to the latest dev commit~ / no protobuf update needed
- [x] `BackendService` unchanged / ~`BackendService` changed and corresponding ICD test fix added~

### CODE REVIEW

**Comment by kswang1029** [comment]
we might consider the timing _when_ we merge this in dev: when this is ready or as the last merge right before the release. It is about solving merge conflict.

File: src/components/Dialogs/FileBrowser/FileBrowserDialogComponent.tsx
Comment (acdo2002) [inline src/components/Dialogs/FileBrowser/FileBrowserDialogComponent.tsx]: ds9 region file?

Comment (kswang1029) [reply src/components/Dialogs/FileBrowser/FileBrowserDialogComponent.tsx]: fixed

File: src/components/Dialogs/FileBrowser/FileBrowserDialogComponent.tsx
Comment (acdo2002) [inline src/components/Dialogs/FileBrowser/FileBrowserDialogComponent.tsx]: sorry I should explain more detail, because "CRTF region" using "file", however "DS9 region" using "File", should these two file being consistant?

Comment (kswang1029) [reply src/components/Dialogs/FileBrowser/FileBrowserDialogComponent.tsx]: also fixed

**Review by acdo2002 — APPROVED** [review]

**Review by confluence — APPROVED** [review]

**Review by YuHsuan-Hwang — CHANGES_REQUESTED** [review]
This is a lot of hard work! Some comments:
1. need to be "Code Snippets" and "Edit Code Snippet"
image
image

2. tooltips of these buttons need to be sentence-case
image
3. Do we set dropdown contents in sentence-case?
image
image
image
4. Does "Scroll" need to be lower-case?
image
image
5. Maybe not related to this issue, but I suggest we change it to "Negative only"
image

**Comment by kswang1029** [comment]
@YuHsuan-Hwang thanks for your eagle eyes~
All are addressed except the level generator modes. Let's consider those as exceptions, unless we have better conventions.

**Review by YuHsuan-Hwang — APPROVED** [review]
> Let's consider those as exceptions, unless we have better conventions.

Sure 👍 

Do you think we need to update change log?

**Comment by kswang1029** [comment]
> > Let's consider those as exceptions, unless we have better conventions.
> 
> Sure 👍
> 
> Do you think we need to update change log?

Not necessary I suppose. This is an embarrassing but harmless “bug”.

**Pull request merged**
